bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,22 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement(
```

Comment:
Can you format it like
```
REQUIREMENTS = {GoRequirement('github.com/BurntSushi/toml/cmd/tomlv',
                              flag='-u')}
```

It is helpful if the first parameter is on the same line as the `*Requirement`

However, the `-u` flag is a bit silly.  That just updates it, which isnt sensible when we have versioning.